### PR TITLE
multisite-setup: Added bash script to simplify the setup process and fixed typo in stylelint configuration.

### PR DESCRIPTION
**Pull request by sandipta18**

### CODE REVIEW

**Comment by sandipta18** [comment]
Please don't merge atm.

**Comment by sandipta18** [comment]
Please review @kunal-sin

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+
+read -p "Enter the site name: " site_name
+read -p "Enter the theme name: " theme_name
+
+
+BASE_DIR="docroot/sites/$site_name/themes/custom/$theme_name"
```
Comment (kunal-sin) [inline setup.sh]: @sandipta18 We are hardcoding `docroot` here but some sites use `web`, `webroot`, etc.

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+ORIGINAL_DIR=$(pwd)
+
+
+lando composer require --dev innoraft/drupal-quality-checker
```
Comment (kunal-sin) [inline setup.sh]: `lando` ?

Some projects don't use `lando` :)

File: setup.sh (hunk 1 above)
Comment (sandipta18) [reply setup.sh]: Updated the script to fetch base directory information from user

File: setup.sh (hunk 2 above)
Comment (sandipta18) [reply setup.sh]: Removed Lando.

File: setup.sh [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  cp "$SRC_GRUMPHP_PATH" "$DEST_GRUMPHP_PATH"
+  echo "Replaced $DEST_GRUMPHP_PATH with $SRC_GRUMPHP_PATH"
+else
+  echo "Source file $SRC_GRUMPHP_PATH not found"
+  exit 1
+fi
+
+
+if [ -f "$DEST_GRUMPHP_PATH" ]; then
+  sed -i "s|bin: \"web/themes/custom/<theme_name>/node_modules/.bin/eslint\"|bin: \"$base_dir/sites/$site_name/themes/custom/$theme_name/node_modules/.bin/eslint\"|" "$DEST_GRUMPHP_PATH"
```
Comment (kunal-sin) [inline setup.sh:67]: `$base_dir` is not required here?

**Pull request merged**